Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -367,9 +365,7 @@ void SetStaticResolvers()
                             .GetResourceModel());
 
                 EntityExtensionsFactory.Instance = Container.Resolve<IEntityExtensionsFactory>();
-
-                DbConfiguration.SetConfiguration(new DatabaseEngineDbConfiguration(Container.Resolve<DatabaseEngine>()));
```

Comment:
What replaces this line of code? Is changing this really necessary for the .NET 8 upgrade?